CONTRIBUTING.md

```diff
@@ -1,12 +1,194 @@
-## Setting up for local driver development
 
-The following instructions are for working with the driver source code. Follow this set up if you intend to contribute to the driver. These steps are similar to those for developing with the driver, but include steps for building and testing locally. 
+First off, thank you for considering contributing to *vertica-nodejs* and helping make it even better than it is today!
 
-1. Clone the repo
-2. If yarn is not already installed, install yarn `npm install -g yarn`
-3. From your workspace root run `yarn` and then `yarn lerna bootstrap`
-4. Ensure you have a Vertica instance running with 
-5. Ensure you have the proper environment variables configured for connecting to the instance (`V_HOST`, `V_PORT`, `V_USER`, `V_PASSWORD`, `V_DATABASE`, `V_BACKUP_SERVER_NODE`)
-6. Run `yarn test` to run all the tests, or run `yarn test` from within an individual package to only run that package's tests
+This document will guide you through the contribution process. There are a number of ways you can help:
 
-If using VS Code, you can install the `Remote - Containers` extension and it will use the configuration under the `.devcontainer` folder to automatically create dev containers, including Vertica.  See [here](https://code.visualstudio.com/docs/remote/containers) for more information on developing in containers using VS Code.  This process will complete steps 3 to 5 above.
\ No newline at end of file
+ - [Bug Reports](#bug-reports)
+ - [Feature Requests](#feature-requests)
+ - [Code Contributions](#code-contributions)
+ 
+# Bug Reports
+
+If you find a bug, submit an [issue](https://github.com/vertica/vertica-nodejs/issues) with a complete and reproducible bug report. If the issue can't be reproduced, it will be closed. If you opened an issue, but figured out the answer later on your own, comment on the issue to let people know, then close the issue.
+
+For issues (e.g. security related issues) that are **not suitable** to be reported publicly on the GitHub issue system, report your issues to [Vertica open source team](mailto:vertica-opensrc@microfocus.com) directly or file a case with Vertica support if you have a support account.
```

Comment:
FYI, we could create templates for bugs and features to ensure we get the information we want for each.  Not necessarily part of this PR, just something to consider.

As an example, see the templates in the Spark Connector:
https://github.com/vertica/spark-connector/tree/main/.github/ISSUE_TEMPLATE

Reply:
I like that idea. I'll try to remember that and point Siting towards the Spark Connector as well in case she wants to use that for Python or go